src/ape_accounts/_cli.py

```diff
@@ -157,6 +157,17 @@ def _import(cli_ctx, alias, import_from_mnemonic, custom_hd_path):
     )
 
 
+@cli.command(short_help="Export an account private key")
+@ape_cli_context()
+@existing_alias_argument()
+def export(cli_ctx, alias):
+    path = _get_container().data_folder.joinpath(f"{alias}.json")
+    account = json.loads(path.read_text())
+    password = click.prompt("Enter password to decrypt account", hide_input=True)
+    private_key = EthAccount.decrypt(account, password)
+    cli_ctx.logger.success(f"Account 0x{account['address']} private key: {private_key.hex()}")
```

Comment:
Optional thing, just for awareness: There is a way to bold things, if you want to bold the key or address or anything.
I think it is like `click.style` or something and it takes some markup args.

Reply:
ok I got this working :)